### PR TITLE
Add GetUserMedia test case for verifying media capture APIs could access

### PR DESCRIPTION
**Pull request by haoyunfeix**
from localhost.

Impacted tests(approved): new 1, update 0, delete 0

Unit test platform: Crosswalk Project for Windows 20.49.525.0

Unit test result summary: pass 1, fail 0, block 0

BUG=CTS-1714

### CODE REVIEW

**Comment by haoyunfeix** [comment]
@Honry PTAL

File: usecase/usecase-webapi-xwalk-tests/samples/GetUserMedia/LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Copyright (c) 2014, The WebRTC project authors. All rights reserved.
```
Comment (Honry) [inline usecase/usecase-webapi-xwalk-tests/samples/GetUserMedia/LICENSE.md]: This license file is unnecessary as there is already a license note in each file you integrate. By the way, where is this sample integrated from?

Comment (haoyunfeix) [inline usecase/usecase-webapi-xwalk-tests/samples/GetUserMedia/LICENSE.md]: It's from https://github.com/webrtc/samples/tree/gh-pages/src/content/getusermedia/gum

**Comment by Honry** [comment]
Who is the author of this sample? Does he come from Intel?

**Comment by haoyunfeix** [comment]
It's from webrtc team and Google.

**Comment by Honry** [comment]
LGTM

**Pull request merged**